test/app.js

```diff
@@ -80,78 +83,88 @@ describe('generator-terra-module:app', function () {
-    });
-
-    it('fills the package.json file with project data', function () {
-      assert.fileContent('packages/terra-clinical-monster-cookies/package.json', 'git+https://github.com/cerner/terra-clinical.git');
-      assert.fileContent('packages/terra-clinical-monster-cookies/package.json', 'https://github.com/cerner/terra-clinical/issues');
-      assert.fileContent('packages/terra-clinical-monster-cookies/package.json', 'WEBPACK_CONFIG_PATH=../../../../terra-clinical-site/webpack.config');
-      assert.fileContent('packages/terra-clinical-monster-cookies/package.json', '../../node_modules/terra-toolkit/lib/scripts');
-      assert.fileContent('packages/terra-clinical-monster-cookies/package.json', `"props-table": "$(cd ..; npm bin)/props-table ./src/MonsterCookies.jsx --out-dir ./docs/props-table",`);
-    });
-
-    it('fills the README file with project data', function () {
-      assert.fileContent('packages/terra-clinical-monster-cookies/README.md', '# Terra Clinical Monster Cookies');
-      assert.fileContent('packages/terra-clinical-monster-cookies/README.md', 'img.shields.io/npm/v/terra-clinical-monster-cookies.svg');
-      assert.fileContent('packages/terra-clinical-monster-cookies/README.md', 'travis-ci.org/cerner/terra-clinical.svg');
-      assert.fileContent('packages/terra-clinical-monster-cookies/README.md', 'npm install terra-clinical-monster-cookies');
-      assert.fileContent('packages/terra-clinical-monster-cookies/README.md', '[Documentation](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-monster-cookies/docs)');
-    });
-
-    it('fills the examples Index file with project data', function () {
-      assert.fileContent('packages/terra-clinical-site/src/examples/monster-cookies/Index.jsx', `import React from 'react';`);
-      assert.fileContent('packages/terra-clinical-site/src/examples/monster-cookies/Index.jsx', `import MonsterCookiesSrc from '!raw-loader!terra-clinical-monster-cookies/src/MonsterCookies';`);
-      assert.fileContent('packages/terra-clinical-site/src/examples/monster-cookies/Index.jsx', `import ReadMe from 'terra-clinical-monster-cookies/docs/README.md';`);
-      assert.fileContent('packages/terra-clinical-site/src/examples/monster-cookies/Index.jsx', `import { version } from 'terra-clinical-monster-cookies/package.json';`);
-      assert.fileContent('packages/terra-clinical-site/src/examples/monster-cookies/Index.jsx', `export default MonsterCookiesExamples;`);
-    });
-
-    it('fills the examples Index file with project data', function () {
-      assert.fileContent('packages/terra-clinical-monster-cookies/.npmignore', `src`);
-      assert.fileContent('packages/terra-clinical-monster-cookies/.npmignore', `node_modules`);
-      assert.fileContent('packages/terra-clinical-monster-cookies/.npmignore', `*.log`);
+  ['terra-clinical', 'terra-consumer'].forEach(repository => {
+    describe(repository, function () {
```

Comment:
Agreed. At this point, the only differences between the repositories would be the template package file-the differences being the props table script and the key words used.